pkg/crc/image/image.go

```diff
@@ -0,0 +1,103 @@
+package image
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"runtime"
+	"strings"
+
+	crcpreset "github.com/code-ready/crc/pkg/crc/preset"
+	"github.com/containers/image/v5/copy"
+	"github.com/containers/image/v5/signature"
+	"github.com/containers/image/v5/transports/alltransports"
+	"github.com/containers/image/v5/types"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+var (
+	registryURI = "quay.io"
+	orgName     = "crcont"
+)
+
```

Comment:
To make this easily work with another registry, we could include the `registryURI` and `orgName` also inside the `OCIImageHandler` struct and create a constructor like function:
```go
func NewOCIImageHandler(orgName, registryURI) *OCIImageHandler {
    return &OCIImageHandler{orgName: orgName, registryURI: registryURI}
}

``` 

but probably we'll only need to use quay for our usecase

Reply:
Yes I thought about it but initially I am just going with quay. These improvements we can have after first iteration.